chore(docs): add custom seed paths documentation (#29512)
supabase/supabase: apps/docs/content/guides/cli/seeding-your-database.mdx

  ('Mexico', 'MX');
```

If you want to manage multiple seed files or organize them across different folders, you can configure additional paths or glob patterns in your `config.toml` (see the [next section](#splitting-up-your-seed-data) for details).

### Splitting up your seed file

For better modularity and maintainability, you can split your seed data into multiple files. For example, you can organize your seeds by table and include files such as `countries.sql` and `cities.sql`. Configure them in `config.toml` like so:

```toml supabase/config.toml
[db.seed]

>   enabled = true
>   sql_paths = ['./countries.sql', './cities.sql']

Reviewer: ```suggestion
enabled = true
sql_paths = ['./countries.sql', './cities.sql']
```

```

Or to include all `.sql` files under a specific folder you can do:

```toml supabase/config.toml
[db.seed]

>   enabled = true
>   sql_paths = ['./seeds/*.sql']

Reviewer: ```suggestion
enabled = true
sql_paths = ['./seeds/*.sql']
```

```

<Admonition type="tip">

> The system processes the seed files in the order they are listed in the sql_paths array. If a glob pattern is used and matches multiple files, those files are sorted in lexicographic order to ensure consistent execution. Additionally:

Reviewer: ```suggestion
The CLI processes seed files in the order they are declared in the `sql_paths` array. If a glob pattern is used and matches multiple files, those files are sorted in lexicographic order to ensure consistent execution. Additionally:
```
